packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -4838,8 +4255,6 @@ test.serial('updateGranuleStatusToQueued() throws error if record does not exist
 test.serial('updateGranuleStatusToQueued() does not update Elasticsearch granule if writing to PostgreSQL fails', async (t) => {
```

Comment:
With es removed, we can test:
updateGranuleStatusToQueued() does not publish an SNS message if writing to PostgreSQL fails